src/applications/mhv-medications/routes.jsx

```diff
@@ -93,7 +93,6 @@ const routes = [
   {
     path: 'prescription/:prescriptionId/documentation',
     element: <AppWrapper Component={PrescriptionDetailsDocumentation} />,
-    loader: authenticatedLoader({ loader: prescriptionsLoader }),
```

Comment:
this avoids re-fetching the prescription data for the normal case when someone is navigating to the documentation page from the prescription details page.